src/PointwiseFunctions/Hydro/Tags.hpp

```diff
@@ -211,4 +213,17 @@ struct MassFlux : db::SimpleTag {
   static std::string name() { return Frame::prefix<Fr>() + "MassFlux"; }
 };
 }  // namespace Tags
+
+/// %Tags for options of hydrodynamic systems.
+namespace OptionTags {
+
+/// The equation of state of the fluid.
+template <size_t ThermoDim = 0, bool IsRelativistic = true>
+struct EquationOfState {
+  using type = std::unique_ptr<
+      EquationsOfState::EquationOfState<ThermoDim, IsRelativistic>>;
```

Comment:
Am I looking at the wrong class, because it looks like the template parameters are in the wrong order?
```c++
template <bool IsRelativistic, size_t ThermodynamicDim>
class EquationOfState;
```

Reply:
No, I think this is a bug which hasn't caused problems because so far this code has only been run with dim= 1